scripts/azure_cleanup.py

```diff
@@ -36,68 +36,32 @@ def azure_cleanup(nic_template, pip_template, days_old):
                 mgmt = get_mgmt(prov_key)
                 mgmt.logger = logger
                 for name, scr_id in mgmt.list_subscriptions():
-                    logger.info("subscription {s} is chosen".format(s=name))
+                    logger.info("Subscription '{s}' is chosen".format(s=name))
```

Comment:
Pass options to logger instead of using format.

`logger.info('Subscription "%s" is chosen', name)`